service/awsconfig/v10/version_bundle.go

```diff
@@ -34,6 +34,11 @@ func VersionBundle() versionbundle.Bundle {
 				Description: "Detach EBS volumes before deletion when deleting clusters.",
 				Kind:        versionbundle.KindChanged,
 			},
+			{
+				Component:   "aws-operator",
+				Description: "Opened port 4194 for Cadvisor scraping from host cluster.",
```

Comment:
Engaging nitpick mode but this should be CAdvisor or cAdvisor which is what they use.

https://github.com/google/cadvisor

Reply:
omg i've been spelling this wrong forever

Reply:
Fixed

Reply:
Thanks! <3 I saw it and then I couldn't unsee it. You'll probably have the same problem now! 😆

Reply:
You have cursed me :(